config/dex.yaml

```diff
@@ -0,0 +1,55 @@
+issuer: '{{ getenv "APP_HOME_URL" }}/dex'
+
+storage:
+  type: sqlite3
+  config:
+    file: /var/dex/dex.db
```

Comment:
Currently in volatile storage

Reply:
Handled now by `$UID/$GID` combo in the container matching the host.